prov/util/src/util_coll.c

```diff
@@ -791,8 +793,16 @@ static int util_coll_find_local_rank(struct fid_ep *ep, struct util_coll_mc *col
 		free(addr);
 		return ret;
 	}
-	coll_mc->local_rank =
-		ofi_av_lookup_fi_addr(coll_mc->av_set->av, addr);
+	my_addr = ofi_av_lookup_fi_addr(coll_mc->av_set->av, addr);
+
+	coll_mc->local_rank = FI_ADDR_NOTAVAIL;
+        if (my_addr != FI_ADDR_NOTAVAIL) {
+		for (i=0; i<coll_mc->av_set->fi_addr_count; i++)
+			if (coll_mc->av_set->fi_addr_array[i] == my_addr) {
+			coll_mc->local_rank = i;
+			break;
```

Comment:
indentation is off.  spacing is also off in places.  Make sure to use tabs for indentation.

Reply:
fixed